### DISCUSSION TITLE
B860h Latest kernel

### DISCUSSION THREAD

**Discussion by diktus** [ophub/amlogic-s9xxx-armbian#2094] [Amlogic]
I have armbian 23.8 running on my zte b860h with kernel 5.9.0 on eMMC. I just check the build is from 150balbes and it can not update the kernel. I try update using apt upgrade. The are kernel version and initram on folder /boot and also on folder /. But when i try uname -a the kernel still show 5.9.0

i try to burn new build Armbian_24.2.0_amlogic_s905x_bookworm_6.6.15_server_2024.02.01.img to sdcard and it won’t boot.

i also try using flashdisk and doest not work. Same with the bullyse image.

what i have try
1. no changing anything just burn to microsd using balena etcher
2.Change boot.ext 905-p212 
3. Add ext.conf

What is the correct way to get working

Thank You

**Comment by amirulshukry** [comment]
Try Armbian_24.2.0_amlogic_s905x_bookworm_6.1.76_server_2024.02.01.img.gz

**Reply by diktus** [reply]
I will try that one. By the way what the different 6.176 with 6.6.15?

**Reply by amirulshukry** [reply]
kernel version

**Reply by amirulshukry** [reply]
In /boot/uEnv.txt change to FDT=/dtb/amlogic/meson-gxl-s905x-b860h.dtb

**Reply by diktus** [reply]
thank you. can boot using Armbian_24.2.0_amlogic_s905x_bookworm_6.1.76_server_2024.02.01.img.gz with change uEnv.txt to meson-gxl-s905x-b860h.dtb. But wifi is not detected. do you know how to enable wifi or is there any DTB that enable wifi on kernel 6.1.76?
i try to use DTB wifi enabled for kernel 5.10 and can not boot using this new armbian version

**Reply by amirulshukry** [reply]
Sorry, my use case did not require wifi so I did not investigate how to turn it on. If you can find a way, perhaps you can share. Good luck.

**Reply by amirulshukry** [reply]
Cursory search on youtube using terms "armbian stb b860h wifi on" shows a few promising videos

**Reply by diktus** [reply]
yes previously with kernel 5.10 i use DTB file that can enable wifi. still looking tutorial to modify DTB to enable wifi on kernel 6.1

**Reply by amirulshukry** [reply]
Have you tried this https://github.com/mregha/Armbian-Wifi/releases ?
